test/e2e/storage/csi_volumes.go

```diff
@@ -92,7 +93,7 @@ var _ = Describe("PMEM Volumes", func() {
 		// testsuites.InitSnapshottableTestSuite,
 		// testsuites.InitSubPathTestSuite,
 		// testsuites.InitVolumeIOTestSuite,
-		// testsuites.InitVolumeModeTestSuite,
```

Comment:
Hmm, did the tests actually run for your in block mode? I only get:
```
PMEM Volumes
/nvme/gopath/src/github.com/intel/pmem-csi/test/e2e/storage/csi_volumes.go:57
  [Driver: pmem-csi]
  /nvme/gopath/src/github.com/intel/pmem-csi/test/e2e/storage/csi_volumes.go:102
    [Testpattern: Dynamic PV (block volmode)] volumeMode
    /nvme/gopath/src/github.com/intel/pmem-csi/vendor/k8s.io/kubernetes/test/e2e/storage/testsuites/base.go:92
      should not mount / map unused volumes in a pod [BeforeEach]
      /nvme/gopath/src/github.com/intel/pmem-csi/vendor/k8s.io/kubernetes/test/e2e/storage/testsuites/volumemode.go:324

      Driver pmem-csi doesn't support  -- skipping

      /nvme/gopath/src/github.com/intel/pmem-csi/vendor/k8s.io/kubernetes/test/e2e/storage/testsuites/base.go:151
```

Reply:
That's because the default file system (i.e. the empty string) is no longer listed as supported:
https://github.com/intel/pmem-csi/commit/1f61a3a426fdfe75f280e17ef8adcab5f8f67225

Reply:
> Hmm, did the tests actually run for your in block mode? I only get:

You are right, I realized this when working on ephemeral volumes branch and fixed here : https://github.com/intel/pmem-csi/pull/383/commits/c7f560cbdf3eb910632b79d66b6f921e173d9372